legwork/evol.py

```diff
@@ -8,7 +8,7 @@
 import astropy.units as u
 import astropy.constants as c
 
-__all__ = ['de_dt', 'get_a_evol', 'get_e_evol', 'get_f_and_e',
```

Comment:
I left `check_mass_freq_input` and `create_timesteps_array` out because I figured they are only really needed internally and people won't really need to use/understand them? My understanding was `__all__` is so that you can do `from evol import *` and in that case I don't think people need these two funcs. Happy to add them if that's better practice or something though!

Reply:
My main thought was to have them included in toc trees. But I don't feel super strongly either way.

Reply:
Ah yeah good point, I'll add them in!